.github/workflows/tests.yaml

```diff
@@ -72,3 +72,24 @@ jobs:
 
             -   name: "Run tests"
                 run: "vendor/bin/phpunit --group e2e"
+
+    static-analysis:
```

Comment:
Might be worth to put in a different workflow file like `phpstan.yaml`?

Reply:
I have a binary brain. Is "might be" a yes or a no?
QA could be 1 workflow.
e.g. https://github.com/szepeviktor/debian-server-tools/blob/master/webserver/Continuous-integration-Continuous-delivery.yml

Reply:
I prefer a different file  pls :)

Reply:
Your wish is my command.